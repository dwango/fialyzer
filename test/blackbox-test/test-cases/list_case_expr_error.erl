-module(list_case_expr_error).

-export([main/1]).

main(Xs) ->
  (fun (Xs) ->
     case Xs of
       [X]  -> X and X
      end
    end)([1,2,3]).
