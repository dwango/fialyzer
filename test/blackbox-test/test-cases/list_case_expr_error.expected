TODO:filename:-1: Type error: type mismatch;
  found   : 'foo'
  required: number()
there is no solution that satisfies subtype constraints
